src/languages/r.coffee

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+
+  name: "R"
+  namespace: "r"
+
+  ###
+  Supported Grammars
+  ###
+  grammars: [
+    "R"
+]
+
+  ###
+  Supported extensions
+  ###
+  extensions: [
+    "r"
+    "R"
```

Comment:
In the future, extensions will be case-insensitive. Thanks for adding both cases in for now, since case-insensitive extensions is not yet supported.